packages/context/src/inject.ts

```diff
@@ -24,6 +24,8 @@ const PROPERTIES_KEY = MetadataAccessor.create<Injection, PropertyDecorator>(
   'inject:properties',
 );
 
+/* eslint-disable @typescript-eslint/ban-types */
```

Comment:
Why are we disabling this rule instead of fixing the linting violations as we do in other files?

Reply:
It does not allow `Object` and there is no configuration to exclude certain types.